docs/reference/modules/node.asciidoc

```diff
@@ -2,30 +2,29 @@
 == Node
 
 Any time that you start an instance of Elasticsearch, you are starting a
-_node_. A collection of connected nodes is called a
-<<modules-cluster,cluster>>. If you are running a single node of Elasticsearch,
-then you have a cluster of one node.
+_node_. A collection of connected nodes is called a <<modules-cluster,cluster>>. 
```

Comment:
Whitespace nit: if we're reformatting this paragraph to the correct width (80 characters) then `_node_.` fits on the previous line.